src/components/tables/BillLineItems/Rows.tsx

```diff
@@ -0,0 +1,39 @@
+import { TableCell, TableRow, Typography } from '@mui/material';
+import { InvoiceLineItem } from 'api/billing';
+import MonetaryValue from 'components/tables/cells/MonetaryValue';
+
+interface RowProps {
+    row: InvoiceLineItem;
+}
+
+interface RowsProps {
+    lineItems: InvoiceLineItem[];
+}
+
+function Row({ row }: RowProps) {
+    return (
+        <TableRow hover>
+            <TableCell>
+                <Typography>{row.description}</Typography>
+            </TableCell>
+            <TableCell>
+                <Typography>{row.count}</Typography>
+            </TableCell>
+
+            <MonetaryValue amount={row.rate / 100} />
+            <MonetaryValue amount={row.subtotal / 100} />
```

Comment:
We seem to always be dividing by `100` when calling this... so I think it is alright to make that a part of the `MonetaryValue` cell itself.